src/routes/LandingPage/SplashSection/index.scss

```diff
@@ -0,0 +1,45 @@
+.SplashSection {
+    height: 772px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+};
+
+.SplashBlurb {
+    font-size: 1rem;
+    color: rgba(25, 4, 49, 1);
+    max-width: 35%;
+    margin-bottom: 100px;
+    margin-left: 100px;
+    text-align: left;
+}
+
+.SplashButton {
```

Comment:
https://stackoverflow.com/questions/11497094/remove-border-from-buttons

Removing that default border would really clean this up